src/routes/articles.js

```diff
@@ -16,15 +17,26 @@ const {
 
 const { slugChecker } = verifySlug;
 
+const articlesBaseEndpoint = '/articles';
+
 articlesRouter
-  .route('/articles')
+  .route(articlesBaseEndpoint)
   .get(getAllArticles)
   .post(verifyToken, checkInput, createArticle);
 
 articlesRouter
-  .route('/articles/:slug')
+  .route(`${articlesBaseEndpoint}/:slug`)
   .get(verifyToken, slugChecker, getArticle)
   .put(verifyToken, slugChecker, checkUser, updateArticle)
   .delete(verifyToken, slugChecker, checkUser, deleteArticle);
 
+articlesRouter
+  .route(`${articlesBaseEndpoint}/:slug/comments`)
+  .get([slugChecker, Comment.getAll])
```

Comment:
The array (square brackets) approach is for loading middlewares in an orderly fashion. Since an array is indexed based, the order in which the middlewares are called 

As for the second question, destructuring the `Comment` methods may clash with someone's method name take for example

```js
import { getAll, create } from '../controllers/comment'
import { getAll, create } from '../controllers/tags'
```
as you can see, we could have another controller that uses the same method name

Using `Comment.[method()]` here means `Comment` act as a namespace for the methods being used

Reply:
Alright then, valid point 👍